docs/section-7/feedforward-control.md

```diff
@@ -0,0 +1,213 @@
+
+Afterwards, put them into the SysID tool which can be opened by `Ctrl + Shift + P` and typing `Start Tool`. Here are the following articles for analyzing the data and determining the constants:
+
+- [Loading Data](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/system-identification/loading-data.html)
+- [Viewing Diagnostics](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/system-identification/viewing-diagnostics.html)
+- [Analyzing Data](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/system-identification/analyzing-gains.html)
+- [Additional Utilities and Tools](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/system-identification/additional-utils.html)
+
+Once you have gotten good data and analysis, you should obtain kS, kV, kA and PID constants.
+
+!!! warning
+    The PID constants are only a starting point and should be tuned more.
+
+## Implementation
+
+WPILIB provides a [SimpleMotorFeedforward](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/controller/SimpleMotorFeedforward.html) class that runs feedforward for a motor. After you obtain your feedforward constants (kS, kV, kA) from SysID, you throw them into the constructor of the `SimpleMotorFeedforward` and use the listed methods.
+```java
+// Create a new SimpleMotorFeedforward with gains kS, kV, and kA
+SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(kS, kV, kA);
+
+// Calculates the feedforward for a velocity of 10 units/second and an 
+// acceleration of 20 units/second^2
+// Units are determined by the units of the gains passed in at construction.
+double volts = feedforward.calculate(10, 20);
+
+// drives the motor to the desired velocity and acceleration calculated 
+// from the feedforward controller
+motor.setVoltage(volts);
+```
+
+That's it!
+Similarly WPILIB provides a `ArmFeedforward` and an `ElevatorFeedforward` class whose only difference from `SimpleFeedforward` is that it accepts a \\(k_{g}\\) value.
+
+## Combining Feedforward and Feedback Control
+
+[Read WPILIB's article](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/controllers/combining-feedforward-feedback.html)
+(It is as easy as just adding them together)
+```java
+// Adds a feedforward to the loop output before sending it to the motor
+motor.setVoltage(pid.calculate(encoder.getDistance(), setpoint) + feedforward);
+```
+
+## Conclusion
+
+Using Feedforward and Feedback, you should now be able to control your mechanisms to go at desired velocities and accerations. However, what if you want your motor to turn to a specific position? (Hint: read next section on trapezoidal control).
```

Comment:
separate the "However..." from the previous sentence. Maybe even put it in a `!!! question` block instead.

Reply:
good idea